pallet/assets-ext/src/lib.rs

```diff
@@ -110,6 +111,8 @@ pub mod pallet {
 		pallet_assets::Config<AssetId = AssetId, Balance = Balance>
 		+ pallet_balances::Config<Balance = Balance, ReserveIdentifier = PalletIdValue>
 	{
+		// Id used for identifying assets.
```

Comment:
Doc comments use 3 slashes
```suggestion
		/// Id used for identifying assets.
```